### PR TITLE
Add height property to the Gaussian2D component

### PR DESCRIPTION
**Pull request by magnunor**
### Progress of the PR
- [x] Added height property
- [x] update docstring
- [x] add entry to `CHANGES.rst`
- [x] add test
- [x] ready for review

### Example
```python
import hyperspy.api as hs
g = hs.model.components2D.Gaussian2D()
print(g.height) # 0.15915494309189535
g.A.value = 100
print(g.height) # 15.915494309189533
g.sigma_x.value = 2
print(g.height) # 7.957747154594767
```

### CODE REVIEW

File: hyperspy/_components/gaussian2d.py [hunk 1]
```diff
@@ -151,3 +153,8 @@ def rotation_major_axis(self):
             return self.rotation_angle_wrapped
         else:
             return self.rotation_angle_wrapped - np.pi / 2
+
+    @property
```
Comment (jlaehne) [inline hyperspy/_components/gaussian2d.py]: Place `height` directly after the FWHM properties?

File: hyperspy/_components/gaussian2d.py [hunk 2]
```diff
@@ -151,3 +153,8 @@ def rotation_major_axis(self):
...
+    def height(self):
```
Comment (jlaehne) [inline hyperspy/_components/gaussian2d.py]: Add a `setter` like in the 1D-Gaussian https://github.com/hyperspy/hyperspy/blob/48bf6cb51786ddc8597c969dd46286d1117f40df/hyperspy/_components/gaussian.py#L200

File: hyperspy/tests/component/test_gaussian2d.py [hunk 3]
```diff
@@ -96,3 +97,15 @@ def test_properties():
     g.rotation_angle.value = angle
     np.testing.assert_allclose(g.rotation_angle_wrapped, angle)
     np.testing.assert_allclose(g.rotation_major_axis, angle - np.pi / 2)
+
+
+def test_height():
```
Comment (jlaehne) [inline hyperspy/tests/component/test_gaussian2d.py]: Do tests in analogy to `fwhm`? i.e. `get`, `set` and `getset`

**Comment by magnunor** [comment]
All done!

**Review by jlaehne — APPROVED** [review]

**Pull request merged**